recipes/libjpeg/all/test_package/CMakeLists.txt

```diff
@@ -10,11 +10,3 @@ if (MSVC)
     target_compile_definitions(${PROJECT_NAME} PRIVATE _CRT_SECURE_NO_WARNINGS _CRT_NONSTDC_NO_WARNINGS)
 endif()
 
-if(BUILD_TRANSUPP)
-    add_executable(test_transupp test_transupp.c ${LIBJPEG_RES_DIR}/transupp.c)
-    target_link_libraries(test_transupp PRIVATE JPEG::JPEG)
```

Comment:
we can probably still test this minimally - this appears to be unconditionally included in the recipe.
perhaps just add an `#include "transupp.h"` in `test_package.c` and that's it.